modules/debts_api/lib/debts_api/v0/fsr_form_transform/income_calculator.rb

```diff
@@ -87,7 +87,7 @@ def get_monthly_income
           {
             vetIncome: vet_income,
             spIncome: sp_income,
-            totalMonthlyNetIncome: total_monthly_net_income
+            totalMonthlyNetIncome: total_monthly_net_income || 0
```

Comment:
My only concern is, is this supposed to be an int or a float?

Feel free to resolve

Reply:
I'll double-check the post-process data today for this.

Reply:
@digitaldrk I updated the PR to include the float since that's what the post-processed data had. I rebased this on `master` again, so it should be good to go now.